android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -2097,6 +2145,64 @@ private void sendReactNativeEvent(final String name, @Nullable final WritableMap
+        constants.put( "ON_INTERSTITIAL_CLICKED_EVENT", ON_INTERSTITIAL_CLICKED_EVENT );
+        constants.put( "ON_INTERSTITIAL_DISPLAYED_EVENT", ON_INTERSTITIAL_DISPLAYED_EVENT );
+        constants.put( "ON_INTERSTITIAL_AD_FAILED_TO_DISPLAY_EVENT", ON_INTERSTITIAL_AD_FAILED_TO_DISPLAY_EVENT );
+        constants.put( "ON_INTERSTITIAL_HIDDEN_EVENT", ON_INTERSTITIAL_HIDDEN_EVENT );
+        constants.put( "ON_INTERSTITIAL_AD_REVENUE_PAID", ON_INTERSTITIAL_AD_REVENUE_PAID );
+
+        constants.put( "ON_REWARDED_AD_LOADED_EVENT", ON_REWARDED_AD_LOADED_EVENT );
+        constants.put( "ON_REWARDED_AD_LOAD_FAILED_EVENT", ON_REWARDED_AD_LOAD_FAILED_EVENT );
+        constants.put( "ON_REWARDED_AD_CLICKED_EVENT", ON_REWARDED_AD_CLICKED_EVENT );
+        constants.put( "ON_REWARDED_AD_DISPLAYED_EVENT", ON_REWARDED_AD_DISPLAYED_EVENT );
+        constants.put( "ON_REWARDED_AD_FAILED_TO_DISPLAY_EVENT", ON_REWARDED_AD_FAILED_TO_DISPLAY_EVENT );
+        constants.put( "ON_REWARDED_AD_HIDDEN_EVENT", ON_REWARDED_AD_HIDDEN_EVENT );
+        constants.put( "ON_REWARDED_AD_RECEIVED_REWARD_EVENT", ON_REWARDED_AD_RECEIVED_REWARD_EVENT );
+        constants.put( "ON_REWARDED_AD_REVENUE_PAID", ON_REWARDED_AD_REVENUE_PAID );
+
+        constants.put( "ON_APPOPEN_AD_LOADED_EVENT", ON_APPOPEN_AD_LOADED_EVENT );
+        constants.put( "ON_APPOPEN_AD_LOAD_FAILED_EVENT", ON_APPOPEN_AD_LOAD_FAILED_EVENT );
+        constants.put( "ON_APPOPEN_AD_CLICKED_EVENT", ON_APPOPEN_AD_CLICKED_EVENT );
+        constants.put( "ON_APPOPEN_AD_DISPLAYED_EVENT", ON_APPOPEN_AD_DISPLAYED_EVENT );
+        constants.put( "ON_APPOPEN_AD_FAILED_TO_DISPLAY_EVENT", ON_APPOPEN_AD_FAILED_TO_DISPLAY_EVENT );
+        constants.put( "ON_APPOPEN_AD_HIDDEN_EVENT", ON_APPOPEN_AD_HIDDEN_EVENT );
+        constants.put( "ON_APPOPEN_AD_REVENUE_PAID", ON_APPOPEN_AD_REVENUE_PAID );
+
+        constants.put( "TOP_CENTER_POSITION", TOP_CENTER );
+        constants.put( "TOP_LEFT_POSITION", TOP_LEFT );
+        constants.put( "TOP_RIGHT_POSITION", TOP_RIGHT );
+        constants.put( "CENTERED_POSITION", CENTERED );
+        constants.put( "CENTER_LEFT_POSITION", CENTER_LEFT );
+        constants.put( "CENTER_RIGHT_POSITION", CENTER_RIGHT );
+        constants.put( "BOTTOM_LEFT_POSITION", BOTTOM_LEFT );
+        constants.put( "BOTTOM_CENTER_POSITION", BOTTOM_CENTER );
+        constants.put( "BOTTOM_RIGHT_POSITION", BOTTOM_RIGHT );
+
+        constants.put( "BANNER_AD_FORMAT_LABEL", MaxAdFormat.BANNER.getLabel() );
+        constants.put( "MREC_AD_FORMAT_LABEL", MaxAdFormat.MREC.getLabel() );
+
+        constants.put( "CONSENT_DIALOG_STATE_UNKNOWN", AppLovinSdkConfiguration.ConsentDialogState.UNKNOWN.ordinal() );
+        constants.put( "CONSENT_DIALOG_STATE_APPLIES", AppLovinSdkConfiguration.ConsentDialogState.APPLIES.ordinal() );
+        constants.put( "CONSENT_DIALOG_STATE_DOES_NOT_APPLY", AppLovinSdkConfiguration.ConsentDialogState.DOES_NOT_APPLY.ordinal() );
```

Comment:
I think we should actually remove the `consentDialogState` API in v5.0.0 since we've deprecated it already.

Reply:
Ok, I'll remove the consentDialogState API.